types/vuelidate/vuelidate.d.ts

```diff
@@ -10,13 +10,16 @@ export const validationMixin: any
 
 // const Validation
 export interface Validation extends Vue {
+    readonly [rule: string]: any // TODO: fix that with boolean
```

Comment:
What does it take to resolve this to-do?

Reply:
I am so stupid, this is useless and i have remove that.